tgui/packages/tgui/interfaces/SpawnPanel/CreateObjectAdvancedSettings.tsx

```diff
@@ -0,0 +1,126 @@
+import React, { useEffect } from 'react';
```

Comment:
```suggestion
import { useEffect } from 'react';
```
React is already added via bundler

Reply:
I'm guessing this was an autoimport when I was doing something with effects